contrib/test/integration/build/seccomp.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: download seccomp
```

Comment:
@lsm5 Can we get this from some repo that we missed enabling?

Reply:
In case you want the -devel packages, I think those are in the CRB (code-ready builder) repo. I don't have the public facing links, but I can check with RCM and get those to you.